solution/backend/content_search/tests/fixtures/sample_files.json

```diff
@@ -12,7 +12,7 @@
         "summary": "Important notes that cannot be public"
     },
     {
-        "document_name": "Policy reference file",
+        "document_name": "Policy reference file fire fire fire",
```

Comment:
why name it fire fire fire?

Reply:
search relevancy.  there is a search in unit test for fire